staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -44,6 +44,9 @@ const (
 type KubeSchedulerConfiguration struct {
 	metav1.TypeMeta `json:",inline"`
 
+	// Parallelism defines the amount of parallelism that should be used in scheduler algorithms and some plugins.
```

Comment:
I would say:

`Parallelism defines the amount of parallelism in algorithms for scheduling a Pod`

This makes it clear that we are NOT scheduling multiple Pods at a time and plugins are internal details that users of this field don't need to know.

Reply:
Please update this

Reply:
Still not updated